spark/src/test/scala/io/delta/sharing/spark/TestDeltaSharingClient.scala

```diff
@@ -97,7 +116,13 @@ class TestDeltaSharingClient(
       ).take(limit.getOrElse(4L).toInt)
     }
 
-    DeltaTableFiles(0, Protocol(0), metadata, addFiles)
+    if (versionAsOf.exists(_ == 1)) {
+      DeltaTableFiles(0, Protocol(0), metadataV1, addFiles)
+    } else if (timestampAsOf.exists(_ == TESTING_TIMESTAMP)) {
+      DeltaTableFiles(0, Protocol(0), metadataV2, addFiles)
```

Comment:
Should these be DeltaTableFiles(1, ...) to make it consistent with getMetadata?

Reply:
@chakankardb  seems that the version is not used.
But you are right, I'll update it.